packages/repository-json-schema/src/build-schema.ts

```diff
@@ -414,6 +414,10 @@ export function modelToJsonSchema<T extends object>(
   debug('Model settings', meta.settings);
 
   const title = buildSchemaTitle(ctor, meta, options);
+  // `title` is the unique identity of a schema,
+  // it should be removed from the `options`
+  // when generating the relation or property schemas
+  if (options.title) delete options.title;
```

Comment:
Maybe clone `options` first?

```ts
const targetOptions = {...options);
delete targetOptions.title;
```

Reply:
@raymondfeng hmm any reason to clone `options`? Since the `title` should not be inherited by any other schemas(relation or property in custom type), I intended to remove the `title` from `options` instead of its copy.

Reply:
It's usually a good idea not to mutate the input object, in this case, `options` as the caller won't expect to assume the argument is changed after the method call.

Reply:
+1 to what Raymond said

Reply:
@raymondfeng @bajtos 👍 thx got it and applied feedback.